Knoema.Client/Search/TimeseriesSearch/TimeseriesSearchLocation.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Knoema.Search.TimeseriesSearch
+{
+	public class TimeseriesSearchLocation
+	{
+		public int DatasetKey { get; set; }
+		public DateTime Expires { get; set; }
+		public string Key { get; set; }
+		public IEnumerable<TimeseriesSearchPositionItem> Position { get; set; }
```

Comment:
No need to make it typed, it can be just an 'Object'.
Then server content of this field could change with no need to update clients.

Reply:
done